generators/server/generator.mjs

```diff
@@ -704,7 +704,7 @@ export default class JHipsterServerGenerator extends BaseApplicationGenerator {
             'java:war': './gradlew bootWar -Pwar -x test -x integrationTest',
             'java:docker': './gradlew bootJar -Pprod jibDockerBuild',
             'java:docker:arm64': 'npm run java:docker -- -PjibArchitecture=arm64',
-            'backend:unit:test': `./gradlew test integrationTest ${excludeWebapp} ${javaCommonLog} ${javaTestLog}`,
```

Comment:
Removing excludeWebapp will cause ci to run frontend tests twice.
Taking something from 6-8 min each run.

You should try to add a dumb controller instead, like imperative+jwt.

Reply:
I added a controller and I put excludeWebapp in order to avoid frontend tests.